tests/integration-tests/tests/storage/test_ebs.py

```diff
@@ -131,6 +132,42 @@ def _test_home_correctly_shared(remote_command_executor, scheduler_commands):
     verify_directory_correctly_shared(remote_command_executor, "/home", scheduler_commands)
 
 
+def _test_ebs_resize(remote_command_executor, mount_dir, volume_size):
+    logging.info("Testing ebs has correct volume size")
+
+    # get the filesystem that the shared_dir is mounted on
+    filesystem_name = remote_command_executor.run_remote_command(
+        "df -h -t ext4 | tail -n +2 |grep '{mount_dir}' | awk '{{print $1}}'".format(mount_dir=mount_dir)
+    ).stdout
+    # example output of "df -h -t ext4"
+    #     Filesystem      Size  Used Avail Use% Mounted on
+    # /dev/nvme1n1p1  9.8G   37M  9.3G   1% /ebs_mount_dir
+    # /dev/nvme2n1p1  9.8G   37M  9.3G   1% /existing_mount_dir
+
+    # get the volume name given the filesystem name
+    volume_name = remote_command_executor.run_remote_command(
+        "lsblk -no pkname {filesystem_name}".format(filesystem_name=filesystem_name)
+    ).stdout
+    # example input: /dev/nvme1n1p1
+    # example output: nvme1n1
```

Comment:
same here: can you move the example output above the code that runs the command?